hbase-server/src/main/java/org/apache/hadoop/hbase/coprocessor/MasterObserver.java

```diff
@@ -508,35 +508,57 @@ default void postAssign(final ObserverContext<MasterCoprocessorEnvironment> ctx,
    * Called prior to unassigning a given region.
    * @param ctx the environment to interact with the framework and master
    * @param regionInfo
-   * @param force whether to force unassignment or not
    */
   default void preUnassign(final ObserverContext<MasterCoprocessorEnvironment> ctx,
-      final RegionInfo regionInfo, final boolean force) throws IOException {}
+    final RegionInfo regionInfo) throws IOException {}
```

Comment:
If we keep both `preUnassign` methods then this default method should call the one that includes a force flag, with the flag set to `false`. That would allow existing coprocessors to be used as-is without recompiling, would avoid needing to make a second call in the `MasterCoprocessorHost`, and would be  easy to document in the javadoc for the deprecation.

Reply:
Will just remove them, since it is IA.LimitedPrivate. Thanks.